cfgov/cfgov/settings/base.py

```diff
@@ -592,6 +592,12 @@
     "ROBOTS_TXT_SEARCH_GOV_ONLY": [("environment is", "beta")],
     # Whether to render auto-generated page translation links.
     "TRANSLATION_LINKS": [],
+    # RAFT Testing
+    "RAFT_TESTING": [
+        ("environment is", "beta"),
+        ("after date", "2023-01-22"),
+        ("before date", "2023-01-31"),
```

Comment:
@mistergone upon running the server after this change, Django outputs this warning:

```
?: (flags.E002) Flag RAFT_TESTING's 'after date' condition has an invalid value.
        HINT: Enter an ISO 8601 date representation.
?: (flags.E002) Flag RAFT_TESTING's 'before date' condition has an invalid value.
        HINT: Enter an ISO 8601 date representation.
```

The flags code [uses a Django function](https://github.com/cfpb/django-flags/blob/86ee7d9d7d169c5f66e3159843b2cec63a875ae6/flags/conditions/conditions.py#L87) to parse the string, which doesn't seem to recognize the date-only format, and returns null:

```
>>> from django.utils import dateparse
>>> dateparse.parse_datetime("2023-01-22")
>>>
```

If you include the time it seems to work properly:

```
>>> dateparse.parse_datetime("2023-01-22T00:00:00")
datetime.datetime(2023, 1, 22, 0, 0)
```